### ISSUE TITLE
Reduce YouTube query frequencies

### ISSUE THREAD

**Issue by Pythonic-Rainbow** [TCLRainbow/DimBot#32]
Google's YouTube API has a query limit: max 10000 queries per day.
Raceline currently exceeded the quota.

**Comment by Pythonic-Rainbow** [comment]
https://www.reddit.com/r/Python/comments/6utoaq/trying_to_get_latest_video_on_youtube_channel/dlw4quh?utm_source=share&utm_medium=web2x&context=3

**Comment by Pythonic-Rainbow** [comment]
That method didn't work at all.

```
https://www.googleapis.com/youtube/v3/search?key=[API KEY]&channelId=[CHANNEL ID]&part=snippet,id&order=date&maxResults=50
```

**Comment by Pythonic-Rainbow** [comment]
https://developers.google.com/youtube/v3/determine_quota_cost

**Comment by Pythonic-Rainbow** [comment]
Check metrics https://console.developers.google.com/apis/api/youtube.googleapis.com/metrics

**Comment by Pythonic-Rainbow** [comment]
Use https://developers.google.com/youtube/v3/docs/activities/list

**Comment by Pythonic-Rainbow** [comment]
Resolved in 9b086fc
